internal/commands/parameters.go

```diff
@@ -74,19 +75,20 @@ func mergeBundleParameters(installation *store.Installation, ops ...mergeBundleO
 			return err
 		}
 	}
-	if err := matchAndMergeParametersDefinition(installation.Parameters, userParams, bndl.Definitions); err != nil {
+	if err := matchAndMergeParametersDefinition(installation.Parameters, userParams, bndl.Definitions, stderr); err != nil {
 		return err
 	}
 	var err error
 	installation.Parameters, err = bundle.ValuesOrDefaults(installation.Parameters, bndl)
 	return err
 }
 
-func matchAndMergeParametersDefinition(currentValues map[string]interface{}, parameterValues map[string]string, parameterDefinitions definition.Definitions) error {
+func matchAndMergeParametersDefinition(currentValues map[string]interface{}, parameterValues map[string]string, parameterDefinitions definition.Definitions, stderr io.Writer) error {
 	for k, v := range parameterValues {
 		definition, ok := parameterDefinitions[k]
 		if !ok {
-			return fmt.Errorf("parameter %q is not defined in the bundle", k)
+			fmt.Fprintf(stderr, "Warning: parameter %q is not defined in the bundle\n", k)
```

Comment:
I wonder if it's not the signal for us to have a real log policy and use something like logrus 🤔 
Obviously this would be a big refactor, and we shouldn't do it in this PR.